webcomponents/input/input.webcomponent.spec.js

```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
     })
 
     it('should set empty text to label when attribute is removed', () => {
-      element.setAttribute('placeholder','test')
+      element.setAttribute('placeholder', 'test')
       element.removeAttribute('placeholder')
       expect(element).to.contain('label').with.text('')
     })
   })
+
+  describe('attribute maxlength', () => {
+    it('should define attribute in child input', () => {
+      element.setAttribute('maxlength', '2')
+      expect(element).to.contain('input').with.attribute('maxlength', '2')
+    })
+
+    it('should update attribute in child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.setAttribute('maxlength', '3')
+      expect(element).to.contain('input').with.attribute('maxlength', '3')
+    })
+
+    it('should remove attribute from child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.removeAttribute('maxlength')
```

Comment:
after to run this line, the attribute is not being excluded, as show below:

https://cloud.githubusercontent.com/assets/23660750/26319344/081b766c-3ef5-11e7-8a3f-f67cc7db95e6.png

And even existing, the test is passing.